library_save.json

```diff
@@ -1,139 +1,37 @@
-      "title": "Elon Musk",
-      "author": "Ашли Ванс"
-    },
-    "Над пропастью во ржи": {
-      "title": "Над пропастью во ржи",
-      "author": "Джером Дэвид Сэлинджер"
-    },
-    "Превращение": {
-      "title": "Превращение",
-      "author": "Франц Кафка"
-    },
-    "Алиса в Стране чудес": {
-      "title": "Алиса в Стране чудес",
-      "author": "Льюис Кэрролл"
-    },
-    "Красное и чёрное": {
-      "title": "Красное и чёрное",
-      "author": "Стендаль"
-    },
-    "Getting Things Done": {
-      "title": "Getting Things Done",
-      "author": "Дэвид Аллен"
-    },
-    "Fluent Forever": {
-      "title": "Fluent Forever",
-      "author": "Габриэл Винер"
-    }
+    "Elon Musk": "#<Book:0x0055c1dea8d360>",
+    "Над пропастью во ржи": "#<Book:0x0055c1dea8a4f8>",
+    "Превращение": "#<Book:0x0055c1dea8a4a8>",
+    "Алиса в Стране чудес": "#<Book:0x0055c1dea8a408>",
+    "Красное и чёрное": "#<Book:0x0055c1dea8a390>",
+    "Getting Things Done": "#<Book:0x0055c1dea8a318>",
+    "Fluent Forever": "#<Book:0x0055c1dea8a2c8>"
   },
   "readers": {
-    "Pit": {
-      "name": "Pit",
-      "email": "pit@gmail.com",
-      "city": "Dnipro",
-      "street": "Gogolya",
-      "house": "655"
-    },
-    "Met": {
-      "name": "Met",
-      "email": "met@gmail.com",
-      "city": "Dnipro",
-      "street": "Svobodi",
-      "house": "452"
-    },
-    "Egor": {
-      "name": "Egor",
-      "email": "egor11@gmail.com",
-      "city": "Dnipro",
-      "street": "Petrova",
-      "house": "66"
-    },
-    "Masha": {
-      "name": "Masha",
-      "email": "mery@gmail.com",
-      "city": "Dnipro",
-      "street": "Zvonka",
-      "house": "194"
-    }
+    "Pit": "#<Reader:0x0055c1dea8a250>",
+    "Met": "#<Reader:0x0055c1dea8a200>",
+    "Egor": "#<Reader:0x0055c1dea8a188>",
+    "Masha": "#<Reader:0x0055c1dea8a138>"
   },
   "orders": [
-    {
-      "book": "Над пропастью во ржи",
-      "reader": "Pit",
-      "date": "2017-11-05 19:17:09 +0200"
-    },
-    {
-      "book": "Над пропастью во ржи",
-      "reader": "Met",
-      "date": "2017-11-05 19:17:09 +0200"
-    },
-    {
-      "book": "Над пропастью во ржи",
-      "reader": "Egor",
-      "date": "2017-11-05 19:17:09 +0200"
-    },
-    {
-      "book": "Алиса в Стране чудес",
-      "reader": "Pit",
-      "date": "2017-11-05 19:17:09 +0200"
-    },
-    {
-      "book": "Алиса в Стране чудес",
-      "reader": "Met",
-      "date": "2017-11-05 19:17:09 +0200"
-    },
-    {
-      "book": "Fluent Forever",
-      "reader": "Egor",
-      "date": "2017-11-05 19:17:09 +0200"
-    },
-    {
-      "book": "Fluent Forever",
-      "reader": "Masha",
-      "date": "2017-11-05 19:17:09 +0200"
-    },
-    {
-      "book": "Getting Things Done",
-      "reader": "Pit",
-      "date": "2017-11-05 19:17:09 +0200"
-    },
-    {
-      "book": "Fluent Forever",
-      "reader": "Met",
-      "date": "2017-11-07 00:01:05 +0200"
-    }
+    "#<Order:0x0055c1dea8a0e8>",
```

Comment:
Looks like there is some problem with saving your objects here) As I see there is just address in memory, but NOT object itself. Have you tested you solution successfully ?